### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by AbrahaKahsay**
## To Do list : list structure
In this project, I will build a simple HTML list of To Do tasks. It is a simple web page that will be built using webpack and served by a webpack dev server.

It has the objectives of using:

- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

## Built with

- Major languages: HTML, CSS, JavaScript
- Technologies used: VS Code, Github, Linters, Firefox web-browser
- webpack

### CODE REVIEW

**Comment by AbrahaKahsay** [comment]
Dear reviewer, 
I couldn't figure out the problem with the Lighthouse. Also, I wanted to update the version but I don't know, how. 
Thank you in advance!

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  {
+    description: 'coding',
+    completed: true,
+    index: 1,
+
+  },
+  {
+    description: 'break dance',
+    completed: false,
+    index: 0,
```
Comment (vechicin) [inline src/index.js]: Make sure you set the right index for this task.

Screen Shot 2022-07-06 at 18 51 39

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  },
+];
+
+// console.log(tasks)
```
Comment (vechicin) [inline src/index.js]: Please remove this commented line of code to keep your codebase organized and clean.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @AbrahaKahsay ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbrahaKahsay** [comment]
@vechicin Thanks for your constructive review. 
Do you have any recommendations on how to fix the Lighthouse linter problem?

**Comment by OmarRamoun** [comment]
@AbrahaKahsay. I believe you forgot to push the code

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @AbrahaKahsay 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors (BTW: the Lighthouse problem is from the source. The lighthouse team introduced a new update the broke the linter)
- The code is top-notch! 💯 and have a huge potential.


Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbrahaKahsay** [comment]
@OmarRamoun Thank you so much for your detailed explanation and inspiration. :1st_place_medal:

**Pull request merged**